doc/userguide/rules/mqtt-keywords.rst

```diff
@@ -46,17 +46,22 @@ Valid values are :
 
 where ``UNASSIGNED`` refers to message type code 0.
 
+mqtt.type uses an :ref:`unsigned 8-bits integer <rules-integer-keywords>`.
+
 Examples::
 
   mqtt.type:CONNECT;
   mqtt.type:PUBLISH;
+  mqtt.type:2;
 
 
 mqtt.flags
 ----------
 
 Match on a combination of MQTT header flags, separated by commas (``,``). Flags may be prefixed by ``!`` to indicate negation, i.e. a flag prefixed by ``!`` must `not` be set to match.
 
+mqtt.flags uses an :ref:`unsigned 8-bits integer <rules-integer-keywords>`
```

Comment:
I'd consider this statement confusing unless also showing an example to illustrate that both negatable flag strings as well as numeric values can be used here. 

Why would someone use the numeric value here? I thought the idea of the rule language is to abstract from literal byte values to semantic identifiers)

Reply:
> Why would someone use the numeric value here? I thought the idea of the rule language is to abstract from literal byte values to semantic identifiers.

You can use either numeric or semantic identifier...
I am not sure it makes sense to use numeric, but it is available

Reply:
Got it.